source/_integrations/tts.markdown

```diff
@@ -14,7 +14,7 @@ ha_platforms:
 ha_integration_type: entity
 ---
 
-Text-to-speech (TTS) enables Home Assistant to speak to you.
+Text-to-speech (TTS) enables Home Assistant to speak to you. This is an internal integration that you cannot use directly. Rather, it is used by [other integrations](https://www.home-assistant.io/integrations/#text-to-speech). If you are using the Home Assistant voice assistant, [Assist](https://www.home-assistant.io/voice_control/), Assist is using TTS when replying to you. Another way to use TTS, is by using [TTS with Home Assistant Cloud](https://www.nabucasa.com/config/tts/).
```

Comment:
Maybe we don't need to repeat this sentence since it's written below in the note about building blocks: "This is an internal integration that you cannot use directly."

Reply:
Thank you for the catch!
#31730